package.json

```diff
@@ -29,47 +29,48 @@
     "@sentry/hub": "7.119.1",
     "@sentry/types": "7.119.1",
     "@sentry/utils": "7.119.1",
-    "@sentry/wizard": "^3.32.0"
+    "@sentry/wizard": "^3.34.2"
   },
   "devDependencies": {
+    "@jest/types": "^26.0.0",
+    "@rollup/plugin-commonjs": "^28.0.1",
+    "@rollup/plugin-node-resolve": "^15.3.0",
+    "@rollup/plugin-terser": "^0.4.4",
+    "@rollup/plugin-typescript": "^12.1.1",
```

Comment:
Since we upgraded the build tooling, is the built shipped JS still the same or where there any changes introduced?